UI/Web/src/app/manga-reader/manga-reader.component.html

```diff
@@ -33,66 +33,68 @@
         </div>
     </ng-container>
 
-    <div (click)="toggleMenu()" class="reading-area" 
-            [ngStyle]="{'background-color': backgroundColor, 'height': readerMode === ReaderMode.Webtoon ? 'inherit' : 'calc(var(--vh)*100)'}" #readingArea>
-        <ng-container *ngIf="readerMode !== ReaderMode.Webtoon; else webtoon">
-            <div class="image-container" [ngClass]="{'d-none': !renderWithCanvas }" [style.filter]="'brightness(' + generalSettingsForm.get('darkness')?.value + '%)'">
-                <canvas #content class="{{getFittingOptionClass()}}"
-                    ondragstart="return false;" onselectstart="return false;">
-                </canvas>
-            </div>
+    <div (touchstart)="onSwipe($event, 'start')" (touchend)="onSwipe($event, 'end')">
```

Comment:
Why do we need another div layer? Will this work on fullscreen since we fullscreen on the readingArea?

Reply:
Wrapping the whole reading area with the event handler seemed to be more effective and less intrusive. Tested working on fullscreen.